packages/rolldown/tests/fixtures/tree-shake/inline-dynamic-imports/_config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineTest } from '@tests'
+import { expect } from 'vitest'
+
+export default defineTest({
+  config: {
+    output: {
+      inlineDynamicImports: false,
```

Comment:
```suggestion
      inlineDynamicImports: true,
```

`inlineDynamicImports` is false by default. I assume you want to test the `true` condition?

Reply:
Nice catch